libs/designsystem/helpers/src/platform.service.ts

```diff
@@ -1,8 +1,7 @@
 import { Injectable } from '@angular/core';
 
-import { DesignTokenHelper } from '@kirbydesign/core';
-
-import { WindowRef } from '../types/window-ref';
+import { WindowRef } from '@kirbydesign/designsystem/types';
+import { DesignTokenHelper } from './design-token-helper';
```

Comment:
Question 1: No, because this is from within the `helpers` entry. Using the suggested import would cause a circular dependency

Question 2: I hope Question answers this